### PR TITLE
Add support for SMART app cards

### PR DESCRIPTION
**Pull request by nschwertner**
For the time being this works for apps based on the SMART JS client launched against the CDS Hooks demo harness' open FHIR API server

### CODE REVIEW

**Comment by jmandel** [comment]
Thanks @nschwertner! This is a nice interim hack. Next step, I think, is to get the test harness running against an off-the-shelf SMART serve instead of an open server (maybe SMART's sandbox, or even our open proxy).

**Comment by kpshek** [comment]
Yes, thanks @nschwertner!

**Comment by kpshek** [comment]
@nschwertner - Can you resubmit this pull request against `master`? I merged the `v-swagger` branch into `master` and then deleted the `v-swagger` branch, not realizing it was going to automatically close this pull request. Sorry! :confused:

File: webpack.dev.config.js [hunk 1]
```diff
@@ -10,34 +11,34 @@ module.exports = {
...
-    new webpack.NoErrorsPlugin(),
+    new webpack.NoEmitOnErrorsPlugin(),
     new webpack.DefinePlugin({
       "runtime.CDS_HOOKS_URL": JSON.stringify(process.env.CDS_HOOKS_URL || "http://localhost:8081"),
       "runtime.FHIR_URL": JSON.stringify(process.env.FHIR_URL || "https://fhir-open-api-dstu2.smarthealthit.org")
     })
   ],
   resolve: {
-    modulesDirectories: ['node_modules'],
+    modules: ['node_modules'],
```
Comment (zplata) [inline webpack.dev.config.js:26]: this change should also be in the webpack prod config file

**Pull request closed without merging**